repose-aggregator/components/filters/client-auth/src/test/groovy/org/openrepose/filters/clientauth/AuthNFilterDeprecationTest.groovy

```diff
@@ -0,0 +1,32 @@
+package org.openrepose.filters.clientauth
+
+import com.mockrunner.mock.web.MockFilterConfig
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.LoggerContext
+import org.apache.logging.log4j.test.appender.ListAppender
+import org.openrepose.core.services.config.ConfigurationService
+import org.openrepose.core.services.datastore.DatastoreService
+import spock.lang.Specification
+
+import static org.mockito.Mockito.mock
+
+class AuthNFilterDeprecationTest extends Specification {
+    def 'logs a deprecation warning upon init'() {
+        given:
+        System.setProperty("javax.xml.parsers.DocumentBuilderFactory",
```

Comment:
So i suspect we are going to broke as crap when live. I have a sneaking suspicion that different parts of the code depend on different libraries now